src/main/java/com/contrastsecurity/maven/plugin/sdkx/Project.java

```diff
@@ -0,0 +1,360 @@
+package com.contrastsecurity.maven.plugin.sdkx;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Objects;
+
+/** Value object that describes a Contrast Scan project */
+public final class Project {
```

Comment:
I'd like to be consistent with the names we use in everywhere else, even if those names are imperfect.

> since that will distinguish it from a local Maven project

I ran into this in the `ContrastScanMojo`. You'll see that I named the `MavenProject` `mavenProject` to distinguish the two. Recall that this type will migrate to the SDK which is unaware of Maven's domain models.